CONTRIBUTING.md

```diff
@@ -16,21 +20,37 @@ To submit a bug report or a feature request, please open an [issue](https://gith
 
 Follow these steps to start contributing code:
 
+1. Setup your environment i.e:
```

Comment:
Maybe we should add this to the dependencies steps? https://github.com/CarperAI/trlx/blob/974d9a975c51805dc8d3ac141090515692657642/CONTRIBUTING.md?plain=1#L29-L33

What do you think?